sites/main-site/src/content/hackathon-projects/hackathon-october-2025/metaomics_shortread_qc_subworkflow.md

```diff
@@ -0,0 +1,28 @@
+---
+title: meta-omics short reads QC
```

Comment:
I dont think this will necessarily be specifc to meta-omics!

You could make it broader to entice more people